### ISSUE TITLE
List of features I want to add

### ISSUE THREAD

**Issue by loskyertt** [hiddify/hiddify-app#874]
### Feature description

1.The user can manually stop the software speed measurement.
2.Added download speed test option(like speedtest).
3.Users can customize routing rules.

**Comment by valo-io** [comment]
yeah I really really hope the speed test gets added. I have multiple subs with lots of proxies, and most of the times, the ones with less delay are not the ones with higher bandwidth, so I have to manually test the speed of each and every one of them till I find the better one. and when I switch to another sub and switch back again, my selection is forgotten, so I have to do the manual tests again.
Having a speed test button would be a very significant QoL feature, and I'm certain everyone that is using this software, would love to have this.

**Comment by WoChen5770** [comment]
I also need custom routing
